contracts/UniswapV3Staker.sol

```diff
@@ -46,6 +46,9 @@ contract UniswapV3Staker is IUniswapV3Staker, Multicall {
         public immutable
         override nonfungiblePositionManager;
 
+    uint256 immutable maxIncentiveDurationSeconds;
```

Comment:
mmmm seconds here and not on the other one is weird... sorry would just remove seconds or add to other, up to you

Reply:
taking out. I feel like seconds is the only standard way to define time anyway